tutorials/2d/using_tilemaps.rst

```diff
@@ -6,160 +6,162 @@ Using tilemaps
 Introduction
 ------------
 
-Tilemaps can be used to make levels for 2D games. 
-A tilemap is a grid of (usually square) image tiles which form the scenery of a level or area in a game. The placed tiles in a tilemap are instances of 'reference tiles' from a tileset. The tileset can be thought of as a pallete with which a tilemap (level) can be 'painted':
+A tilemap is a grid of tiles used to create a game's layout. There are several
+benefits to using a :ref:`TileMap <class_TileMap>` node to design your levels.
+First, they make it possible to draw the layout by "painting' the tiles onto a
```

Comment:
Nitpick: IMO either use plural everywhere: `benefits to using TileMap nodes`, `they make it possible` or use singular consistently: `it makes it possible`. I'd prefer the first.

Reply:
Good point. That one was particularly awkward.